### PR TITLE
Feature/pelagos 3565 add funding org facet

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Resources/public/css/search.css [hunk 1]
```diff
@@ -121,3 +121,9 @@ h1{
     display:inline-block;
 }
 
+#resgrp-facet{
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/public/css/search.css]: a class name would be better, then you can apply it to all facets?

File: src/Pelagos/Bundle/AppBundle/Controller/UI/SearchPageController.php [hunk 2]
```diff
@@ -62,7 +65,8 @@ private function getRequestParams(Request $request): array
             'query' => $request->get('query'),
             'page' => $request->get('page'),
             'options' => array(
-                'rgId' => $request->get('rgId')
+                'rgId' => ($request->get('resGrp')) ? str_replace('rg_', '', $request->get('resGrp')) : null,
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Controller/UI/SearchPageController.php]: Why the value prefixes?

File: src/Pelagos/Bundle/AppBundle/Resources/views/Search/results.html.twig [hunk 3]
```diff
@@ -16,24 +16,47 @@
...
 
-                <ul class="browse-list list-unstyled long-list" style="margin-bottom: 0;">
-                    <li for="{{ group.id }}">
-                        <div class="checkbox block" style="margin-top: 0; margin-bottom: 0;">
-                            <label>
-                                <input class="rg-checkbox" type="checkbox" id="{{ group.id }}">
+                        <li for="{{ group.id }}">
+                            <div class="checkbox block" style="margin-top: 0; margin-bottom: 0;">
+                                <label>
+                                    <input class="rg-checkbox" type="checkbox" id="rg_{{ group.id }}">
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Search/results.html.twig]: Have you considered attributes, or data?

File: src/Pelagos/Bundle/AppBundle/Resources/public/css/search.css (hunk 1 above)
Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Resources/public/css/search.css]: I'm not sure if other facets will have really long lists. I think we wouldn't want to have scrolls for all facets

File: src/Pelagos/Bundle/AppBundle/Controller/UI/SearchPageController.php (hunk 2 above)
Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Controller/UI/SearchPageController.php]: For uniqueness

**Pull request merged**